### PR TITLE
Fixed #34208 -- Confirmed support for GDAL 3.6.

### PR DESCRIPTION
**Pull request by felixxm**
ticket-34208

### CODE REVIEW

File: docs/ref/contrib/gis/install/geolibs.txt [hunk 1]
```diff
@@ -219,27 +220,25 @@ First download the latest GDAL release version and untar the archive::
...
+    $ mkdir build
+    $ cd build
 
-.. note::
+Finally, configure, make and install GDAL::
 
-    Because GeoDjango has its own Python interface, the preceding instructions
-    do not build GDAL's own Python bindings. The bindings may be built by
-    adding the ``--with-python`` flag when running ``configure``. See
-    `GDAL/OGR In Python`__ for more information on GDAL's bindings.
```
Comment (felixxm) [inline docs/ref/contrib/gis/install/geolibs.txt:236]: This note was outdated, see [GDAL docs](https://gdal.org/development/building_from_source.html#cmdoption-arg-BUILD_PYTHON_BINDINGS-BOOL).

File: docs/ref/contrib/gis/install/geolibs.txt [hunk 2]
```diff
@@ -219,27 +220,25 @@ First download the latest GDAL release version and untar the archive::
 
     $ wget https://download.osgeo.org/gdal/X.Y.Z/gdal-X.Y.Z.tar.gz
     $ tar xzf gdal-X.Y.Z.tar.gz
-    $ cd gdal-X.Y.Z
 
-Configure, make and install::
+For GDAL 3.6.x and greater, releases only support builds using ``CMake``. To
```
Comment (felixxm) [inline docs/ref/contrib/gis/install/geolibs.txt:224]: See [release notes](https://github.com/OSGeo/gdal/blob/v3.6.0/NEWS.md#in-a-nutshell): _"CMake is the only build system available in-tree. autoconf and nmake build systems have been removed"_.

**Review by ngnpope — APPROVED** [review]

**Review by smithdc1 — APPROVED** [review]
Looks good to me. I checked with:


```
gdal                      3.6.1          
geos                      3.11.1          
libspatialite             5.0.1           
libsqlite                 3.40.0     
proj                      9.1.1
```

**Comment by felixxm** [comment]
Thanks both :fireworks:

**Pull request merged**